tests/for_udp/quic_transport.toml

```diff
@@ -0,0 +1,33 @@
+[client]
+remote_addr = "127.0.0.1:2332" 
+default_token = "default_token_if_not_specify" 
+
+[client.transport]
+type = "quic"
+[client.transport.tls]
+trusted_root = "examples/tls/test_ca.pem"
+hostname = "testserver"
+
+[client.services.echo] 
+type = "udp"
+local_addr = "127.0.0.1:8080" 
+[client.services.pingpong] 
+type = "udp"
+local_addr = "127.0.0.1:8081" 
+
+[server]
+bind_addr = "0.0.0.0:2332" 
+default_token = "default_token_if_not_specify" 
+
+[server.transport]
+type = "quic"
+[server.transport.tls]
```

Comment:
It's good to reuse code like TlsConfig. But it may be not clear to specify type "quic" and then specify block `transport.tls`. I suggest using a separate block `transport.quic`. This can be done by adding ` pub quic: Option<TlsConfig>` in `TransportConfig`

Reply:
Thing is, QUIC explicitly uses TLS1.3

Reply:
> Thing is, QUIC explicitly uses TLS1.3

But I think if the transport uses quic but the config uses the word `tls` it might confuse the user?

Reply:
> Thing is, QUIC explicitly uses TLS1.3

Yes. That's true. But not clear at the first sight for end users. For now, types and block names are aligned so it's clear how to configure protocols and use them.

Reply:
reused as `transport.quic`